README.md

```diff
@@ -162,6 +162,14 @@ $.Date :: Type
 
 Type comprising every Date value.
 
+#### `Enum`
+
+```haskell
+$.Enum :: [Any] -> Type
+```
+
+An enumerated type comprising a finite set of values.
```

Comment:
`$.Enum` is a type *constructor*. We can consult the description of `$.Array` for guidance:

> Constructor for homogeneous Array types.

Perhaps the following is sufficient?

> Constructor for [enumerated types][1].


[1]: https://en.wikipedia.org/wiki/Enumerated_type